test/eslint-path-test.js

```diff
@@ -40,6 +46,25 @@ describe('eslint-path', () => {
       assert.equals(result, '/some/eslint');
     });
 
+    describe('when ESLINT_D_LOCAL_ESLINT_ONLY is enabled', () => {
+      it('should not resolve a global prettier version', () => {
+        process.env.ESLINT_D_LOCAL_ESLINT_ONLY = 1;
+
+        sinon.replace(resolver, 'resolve', sinon.fake((_, options) => {
+          if (options) {
+            throw new Error('Module not found');
+          }
+          return '/some/eslint';
+        }));
```

Comment:
If I understand correctly, this will be called just once and it needs to throw for the test case. So why not this?
```js
sinon.replace(resolver, 'resolve', sinon.fake.throws(new Error('Module not found')));
```